great-idea-website/css/index.css

```diff
@@ -63,4 +63,72 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+/*ENTIRE PAGE*/
+.container {
+    margin:auto;
+	width: 900px;
+	text-align: left;
+	}
+
+/*NAV @ TOP OF PAGE*/
+header nav {
+	display: inline-block;
+	margin: 50px 0 0 0;
+}
+/*GREAT IDEA LIGHTBULB*/
+header .logo{
+	margin: -25px 140px 2px 637px;
```

Comment:
Here it looks like we shouldn't need negative margins.  We should be able to manage the page without this being done.